src/scripts/main.js

```diff
@@ -2,7 +2,31 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (let i = 0; i < people.length; i++) {
+  const row = document.createElement('tr');
 
-// write your code here
+  const cellName = document.createElement('td');
+  const cellSex = document.createElement('td');
+  const cellBorn = document.createElement('td');
+  const cellDied = document.createElement('td');
+  const cellAge = document.createElement('td');
+  const cellCentury = document.createElement('td');
+
+  cellName.innerHTML = people[i].name;
+  cellSex.innerHTML = people[i].sex;
+  cellBorn.innerHTML = people[i].born;
+  cellDied.innerHTML = people[i].died;
+  cellAge.innerHTML = people[i].died - people[i].born;
+  cellCentury.innerHTML = Math.ceil(people[i].died / 100);
+
+  row.append(cellName);
+  row.append(cellSex);
+  row.append(cellBorn);
+  row.append(cellDied);
+  row.append(cellAge);
+  row.append(cellCentury);
```

Comment:
Do you notice some repetition and similarity of code?
Let's optimize and use the loop or helper function